spec/unleash/strategy/email_from_domain_spec.rb

```diff
@@ -1,6 +1,7 @@
 # frozen_string_literal: true
 
 require 'unleash/context'
+require 'unleash/strategy/email_from_domain'
```

Comment:
Out of curiosity, were these missing `require` preventing specs from running?

Reply:
Yes 😞